packages/message-kit/src/helpers/types.ts

```
@@ -87,15 +78,30 @@ export interface Frame {
   buttons: { content: string; action: string; target: string }[];
   image: string;
 }
+export interface Vibe {
+  name: string;
+  description: string;
+  tone: string;
+  style: string;
+}
+export type SkillHandler = (
+  context: XMTPContext,
+) => Promise<SkillResponse | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)